### PR TITLE
resource/alicloud_cms_site_monitor: Supported interval set to 30, 60

### PR DESCRIPTION
**Pull request by MrWolong**

### CODE REVIEW

File: alicloud/resource_alicloud_cms_site_monitor.go [hunk 1]
```diff
@@ -51,7 +55,7 @@ func resourceAlicloudCmsSiteMonitor() *schema.Resource {
 				Type:         schema.TypeInt,
 				Optional:     true,
 				Default:      1,
-				ValidateFunc: validation.IntInSlice([]int{1, 5, 15}),
+				ValidateFunc: validation.IntInSlice([]int{1, 5, 15, 30, 60}),
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_cms_site_monitor.go:58]: validation 去掉

File: website/docs/r/cms_site_monitor.html.markdown [hunk 2]
```diff
@@ -38,21 +38,41 @@ The following arguments are supported:
...
 ## Attributes Reference
 
 The following attributes are exported:
 
 * `id` - The ID of the site monitor rule.
+* `task_state` - The status of the site monitoring task.
+* `create_time` - The time when the site monitoring task was created.
+* `update_time` - The time when the site monitoring task was updated.
+
+#### Timeouts
```
Comment (xiaozhu36) [inline website/docs/r/cms_site_monitor.html.markdown:62]: Timeouts 前面改成两个井号

**Review by xiaozhu36 — CHANGES_REQUESTED** [review]

File: alicloud/resource_alicloud_cms_site_monitor.go (hunk 1 above)
Comment (MrWolong) [reply alicloud/resource_alicloud_cms_site_monitor.go]: 已修复

File: website/docs/r/cms_site_monitor.html.markdown (hunk 2 above)
Comment (MrWolong) [reply website/docs/r/cms_site_monitor.html.markdown]: 已修复

**Review by xiaozhu36 — APPROVED** [review]

**Pull request merged**